cranelift/filetests/filetests/isa/x86/windows_fastcall_x64.clif

```diff
@@ -8,29 +8,57 @@ function %one_arg(i64) windows_fastcall {
 block0(v0: i64):
     return
 }
-; check: function %one_arg(i64 [%rcx], i64 fp [%rbp]) -> i64 fp [%rbp] windows_fastcall {
-; nextln: ss0 = incoming_arg 16, offset -48
+; check:  function %one_arg(i64 [%rcx], i64 fp [%rbp]) -> i64 fp [%rbp] windows_fastcall {
+; nextln: ss0 = incoming_arg 16, offset -16
+; check:  block0(v0: i64 [%rcx], v1: i64 [%rbp]):
+; nextln: x86_push v1
+; nextln: copy_special %rsp -> %rbp
+; nextln: v2 = x86_pop.i64
+; nextln: return v2
+; nextln: }
 
 ; check if we still use registers for 4 arguments
 function %four_args(i64, i64, i64, i64) windows_fastcall {
 block0(v0: i64, v1: i64, v2: i64, v3: i64):
     return
 }
-; check: function %four_args(i64 [%rcx], i64 [%rdx], i64 [%r8], i64 [%r9], i64 fp [%rbp]) -> i64 fp [%rbp] windows_fastcall {
+; check:  function %four_args(i64 [%rcx], i64 [%rdx], i64 [%r8], i64 [%r9], i64 fp [%rbp]) -> i64 fp [%rbp] windows_fastcall {
+; nextln: ss0 = incoming_arg 16, offset -16
+; check:  block0(v0: i64 [%rcx], v1: i64 [%rdx], v2: i64 [%r8], v3: i64 [%r9], v4: i64 [%rbp]):
```

Comment:
Likewise, I'm not sure what the checks on this function body are intending to test past the `ss0` line?

Reply:
See reply above.  Do you think it would be best if we simply limited it to just the stack layout and prologue/epilogue instructions?

Reply:
(as above, now convinced that these are Good Actually)